packages/manager/apps/pci-volume-backup/src/data/hooks/useVolumeBackups.tsx

```diff
@@ -148,7 +148,7 @@ export function useVolumeBackups<T extends Record<string, unknown>>({
     flattenData: flattenData?.slice(0, (pageIndex + 1) * pageSize),
```

Comment:
Please, do not use these kinds of computations, but rather use a select that flattens data